config/crd/bases/ramendr.openshift.io_drclusters.yaml

```diff
@@ -214,12 +191,7 @@ spec:
                             - Unknown
                             type: string
                           type:
-                            description: |-
-                              type of condition in CamelCase or in foo.example.com/CamelCase.
-                              ---
-                              Many .condition.type values are consistent across resources like Available, but because arbitrary conditions can be
-                              useful (see .node.status.conditions), the ability to deconflict is important.
-                              The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
+                            description: type of condition in CamelCase or in foo.example.com/CamelCase.
```

Comment:
@ShyamsundarR Is this the tool deciding to be concise about the types? The descriptions have become shorter.

Reply:
Yes, instead of the long string it now emits a shorter version of the same.